trackstop.lua

```diff
@@ -0,0 +1,259 @@
+-- Overlay to allow changing track stop friction and dump direction after construction
+--@ module = true
+local gui = require('gui')
+local widgets = require('gui.widgets')
+local overlay = require('plugins.overlay')
+
+local NORTH = 'North'
+local EAST = 'East'
+local SOUTH = 'South'
+local WEST = 'West'
+
+local LOW = 'Low'
+local MEDIUM = 'Medium'
+local HIGH = 'High'
+local HIGHER = 'Higher'
+local MAX = 'Max'
+
+local NONE = 'None'
+
+local FRICTION_MAP = {
+  [NONE] = 10,
+  [LOW] = 50,
+  [MEDIUM] = 500,
+  [HIGH] = 10000,
+  [MAX] = 50000,
+}
+
+local FRICTION_MAP_REVERSE = {}
+for k, v in pairs(FRICTION_MAP) do
+  FRICTION_MAP_REVERSE[v] = k
+end
+
+local SPEED_MAP = {
+  [LOW] = 10000,
+  [MEDIUM] = 20000,
+  [HIGH] = 30000,
+  [HIGHER] = 40000,
+  [MAX] = 50000,
+}
+
+local SPEED_MAP_REVERSE = {}
+for k, v in pairs(SPEED_MAP) do
+  SPEED_MAP_REVERSE[v] = k
+end
+
+local DIRECTION_MAP = {
+  [NORTH] = df.screw_pump_direction.FromSouth,
+  [EAST] = df.screw_pump_direction.FromWest,
+  [SOUTH] = df.screw_pump_direction.FromNorth,
+  [WEST] = df.screw_pump_direction.FromEast,
+}
+
+local DIRECTION_MAP_REVERSE = {}
+for k, v in pairs(DIRECTION_MAP) do
+  DIRECTION_MAP_REVERSE[v] = k
+end
+
+TrackStopOverlay = defclass(TrackStopOverlay, overlay.OverlayWidget)
+TrackStopOverlay.ATTRS{
+  default_pos={x=-71, y=29},
+  default_enabled=true,
```

Comment:
since this is a gameplay mod that lets you do things that are not normally possible, I'm hesitant to enable it by default. Instead, I think you should document in the help that it needs to be turned on in the Overlays tab in `gui/control-panel`.

@ab9rf @lethosor what do you think? is this the right call?

Reply:
if all this does is enable reconfiguring a track stop without reconstructing it, i don't have a problem with it being on by default

Reply:
Ok, how about removing the `gameplay` tag, then. I'd like to keep a clear policy that "gameplay mods are not on by default", and if we're classifying this as "on by defaultable", then I'd like it to not be in the "gameplay" group.